package.json

```diff
@@ -103,15 +103,15 @@
     "eslint": "^7.32.0",
     "eslint-plugin-import": "^2.24.2",
     "eslint-plugin-jsdoc": "^36.0.8",
-    "jest": "^26.6.3",
+    "jest": "^27.5.1",
```

Comment:
I'm not sure jest needed updating, but I had some work around that already, so I decided to keep it.

Reply:
👍